best_practices/forms.rst

```diff
@@ -129,6 +129,10 @@ some developers configure form buttons in the controller::
             ));
 
             // ...
+
+            return $this->render('AcmeTaskBundle:Default:new.html.twig', array(
```

Comment:
The template path should be changed to use the modern Symfony practices:

Instead of this:

```php
return $this->render('AcmeTaskBundle:Default:new.html.twig', array(
```

Use this:

```php
return $this->render('default/new.html.twig', array(
```

Thanks.